crates/biome_formatter/src/lib.rs

```diff
@@ -196,15 +197,19 @@ impl std::fmt::Display for LineEnding {
     }
 }
 
-#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[derive(Clone, Copy, Debug, Eq, Merge, Hash, PartialEq)]
 #[cfg_attr(
     feature = "serde",
-    derive(serde::Serialize, serde::Deserialize, schemars::JsonSchema),
+    derive(serde::Serialize, schemars::JsonSchema),
     serde(rename_all = "camelCase")
 )]
 pub struct IndentWidth(u8);
 
 impl IndentWidth {
+    pub const MIN: u8 = 2;
```

Comment:
Could not allow `0`?

Reply:
That's the bug we're trying to fix 😅

Reply:
I mean, we could default to `2` and allows `0` as value? Users could expect to use `0` to remove any indentation or even `1`.

Reply:
`0` could mean no indentation at all.